app/models/pixelz/modification_request.rb

```diff
@@ -10,16 +10,22 @@ def self.modify(image)
         modifiable_type: image.class.name, modifiable_id: image.id
       ).empty?
       res = Pixelz::ApiClient::Image.new(image).post
-      raise_pixelz_error(res)
+      raise_pixelz_error(res, 'AddImageResponse')
       image.modification_requests
         .create(pixelz_ticket: res['AddImageResponse']['ImageTicket'])
     end
 
-    def self.raise_pixelz_error(res)
-      return if res['AddImageResponse']['ErrorCode'] == 'NoError'
-      raise Pixelz::Error.new res['AddImageResponse']['Message']
+    def self.raise_pixelz_error(res, key)
+      return if res[key]['ErrorCode'] == 'NoError'
+      raise Pixelz::Error.new res[key]['Message']
     end
-    private_class_method :raise_pixelz_error
 
+    def reject(comment)
+      res = Pixelz::ApiClient::Image.new(modifiable).reject(pixelz_ticket, comment)
+      Pixelz::ModificationRequest.send(:raise_pixelz_error, res, 'RejectImageResponse')
+      res
```

Comment:
A few comments on this:

1. Lines are too long. Should not be beyond 80 chars.
2. Why use `send` instead of calling `raise_pixelz_error` directly?
3. Why return `res`, is it used elsewhere?